imports/ui/unit/unit.jsx

```diff
@@ -278,37 +241,51 @@ class Unit extends Component {
-                      >
-                        Assigned To Me
+                    { caseList.length ? (
+                      <div>
+                        <div className='flex flex-grow'>
+                          <SelectField
+                            multiple
+                            hintText='View: All Cases'
+                            value={statusFilterValues}
+                            onChange={this.handleFilterClicked}
+                            autoWidth
+                            underlineStyle={noUnderline}
+                            hintStyle={sortBoxInputStyle}
+                            iconStyle={selectInputIconStyle}
+                            labelStyle={sortBoxInputStyle}
+                            selectedMenuItemStyle={selectedItemStyle}
+                          >
+                            {this.filterMenu(['Open', 'Closed', 'Assigned To Me'], statusFilterValues)}
+                          </SelectField>
+                          <SelectField
+                            hintText='Sort by: Date Added'
+                            value={sortBy}
+                            onChange={this.handleSortClicked}
+                            underlineStyle={noUnderline}
+                            hintStyle={sortBoxInputStyle}
+                            iconStyle={selectInputIconStyle}
+                            labelStyle={sortBoxInputStyle}
+                            selectedMenuItemStyle={selectedItemStyle}
+                          >
+                            {this.sortMenu(sortBy)}
+                          </SelectField>
+                        </div>
+                        { filteredCases.map(caseItem => (
+                          <CaseMenuItem
+                            key={caseItem.id}
+                            className='ph3'
+                            caseItem={caseItem}
+                            onClick={() => {
+                              dispatch(storeBreadcrumb(rootMatch.url))
+                              dispatch(push(`/case/${caseItem.id}`))
+                            }}
+                          />
+                        ))
+                        }
                       </div>
-                    </div>
-                    {filteredCases.map(caseItem => (
-                      <CaseMenuItem
-                        key={caseItem.id}
-                        className='ph3'
-                        caseItem={caseItem}
-                        onClick={() => {
-                          dispatch(storeBreadcrumb(rootMatch.url))
-                          dispatch(push(`/case/${caseItem.id}`))
-                        }}
-                      />
-                    ))}
+                    ) : (<NoItem item='case' iconType='card_travel' />)
+                    }
                   </div>
                   <div className='flex-grow bg-very-light-gray'>
```

Comment:
Same here

Reply:
the old code is replaced by filter & sorter components :)